Kickstarter-iOS/Views/Controllers/VideoViewController.swift

```diff
@@ -152,7 +152,7 @@ public final class VideoViewController: UIViewController {
     }
   }
 
-  func addCompletionObserver(atTime time: CMTime) {
+  func addCompletion(atTime time: CMTime) {
```

Comment:
I would say that this should still have `Observer` in the name, the function is adding the completion observer.